integ/components/deadline/deadline_03_workerFleetHttp/test/deadline_03_workerFleetHttp.test.ts

```diff
@@ -123,7 +123,8 @@ describe.each(testCases)('Deadline WorkerFleet tests (%s)', (_, id) => {
       };
       return awaitSsmCommand(bastionId, params).then( response => {
         var responseOutput = response.output;
-        expect(responseOutput).toMatch(/testpool\ntestgroup\ntestregion/);
+        // Starting Deadline 10.1.11 regions that wasn't added do not apply to worker and returned as unrecognized.
+        expect([/testpool\ntestgroup\nunrecognized/,/testpool\ntestgroup\ntestregion/]).toContain(responseOutput);
```

Comment:
This is inverted. It *works* but a better way to do it would be:
```js
expect(responseOutput).toMatch(/testpool\ntestgroup\n(?:unrecognized|testregion)/);
```

The leading & trailing '/' denote a regex.

You can use the [typescript playground](https://www.typescriptlang.org/play) to experiment, and convince yourself that works:
```js
var ss1 = "testpool\ntestgroup\nunrecognized";
var ss2 = "testpool\ntestgroup\ntestregion";
var ss3 = "testpool\ntestgroup\n";
var ss4 = "testpool\ntestgroup\nfoo";
var regex = /testpool\ntestgroup\n(?:unrecognized|testregion)/;

console.log(regex.test(ss1));
console.log(regex.test(ss2));
console.log(regex.test(ss3));
console.log(regex.test(ss4));
```

Reply:
Yes, your solution looks neatly.